j2ee.common/src/org/netbeans/modules/j2ee/common/dd/resources/beans-1.0.xml

```diff
@@ -1,4 +1,26 @@
 <?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+
+    Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+-->
+
```

Comment:
It seems that this file is a template file.
https://github.com/apache/incubator-netbeans/blob/c8c697418cc0bef6ba070b3e157b5b67cc4080a4/j2ee.common/src/org/netbeans/modules/j2ee/common/dd/DDHelper.java#L139-L157

The other files as well.